src/components/pageLayout/pageHeader.js

```diff
@@ -14,13 +14,26 @@ import { translate } from '../i18n/i18n';
  *
  * @param {object} props
  * @param {Node} props.children
+ * @param {boolean} props.includeTour
  * @param {string} props.productLabel
  * @param {Function} props.t
  * @returns {Node}
  */
-const PageHeader = ({ children, t, productLabel }) => (
+const PageHeader = ({ children, t, productLabel, includeTour }) => (
   <RcsPageHeader>
-    <PageHeaderTitle title={children} className="pf-u-mb-sm" />
+    <Split>
+      <SplitItem>
+        <PageHeaderTitle title={children} className="pf-u-mb-sm" />
+      </SplitItem>
+      <SplitItem isFilled />
+      <SplitItem>
+        {includeTour && (
+          <Button variant="link" className="uxui-curiosity__button-tour">
+            <PfLabel color="blue">{t('curiosity-optin.buttonTour')}</PfLabel>
+          </Button>
+        )}
+      </SplitItem>
+    </Split>
```

Comment:
required/optional: 

First up this totes works as intended!  I feel bad for even asking for this update, especially since the graph is a nightmare on smaller screens.

As you've discovered in PF there are multiple ways to do the same exact thing, kinda... Although we don't really plan for them the `Split` component doesn't appear to stack at smaller sizes, instead it bunches up the title... feels like a prop fix that could be applied to that particular PF component... not sure why they didn't just wrap the `Flex` inside of it anyway other than older browser support maybe... 

> If `Split` is fixed, or there's something we missed, letting it be stacked at smaller screen sizes we have no problem letting this go and sticking with the choice as is!

Lets flip to using the `Flex` and `FlexItem` components here, we already use them on OptIn, so nothing new... they are tricky to use without opening the code up and looking through the props' settings, trying to read through the PF documentation can be frustrating, just go right to the source
- https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/layouts/Flex/Flex.tsx#L257

```
<Flex justifyContent={{ sm: 'justifyContentSpaceBetween' }}>
      <FlexItem>
        <PageHeaderTitle title={children} className="pf-u-mb-sm" />
      </FlexItem>
      <FlexItem>
        {includeTour && (
          <Button variant="link" className="uxui-curiosity__button-tour">
            <PfLabel color="blue">{t('curiosity-optin.buttonTour')}</PfLabel>
          </Button>
        )}
      </FlexItem>
    </Flex>
```

Reply:
Okay, it should be fixed.  I also updated the button to have the isInline prop, so that it wouldn't have button shaped margin around it and would left align all the way.